tools/simple_hydrog/libfmslite/mpp.F90

```diff
@@ -1,3 +1,22 @@
+!***********************************************************************
+!*                   GNU Lesser General Public License
+!*
+!* This file is part of the GFDL FRE NetCDF tools package (FRE-NCTools).
+!*
+!* FRE-NCTools is free software: you can redistribute it and/or modify it under
+!* the terms of the GNU Lesser General Public License as published by
+!* the Free Software Foundation, either version 3 of the License, or (at
+!* your option) any later version.
+!*
+!* FRE-NCTools is distributed in the hope that it will be useful, but WITHOUT
+!* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+!* FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+!* for more details.
+!*
+!* You should have received a copy of the GNU Lesser General Public
+!* License along with FRE-NCTools (LICENSE.md) .  If not, see
+!* <http://www.gnu.org/licenses/>.
+!***********************************************************************
```

Comment:
Still have the old license header. Should this remain an FMS license as the file originated there?

Reply:
In general, it should retain the FMS license information and include a link where the code originated (URL if possible).  However, as we are the _owners_ of the code we are free to change the name in the license header.

Reply:
Rusty,  I believe my process had an error and I did not deal with cases like as I had planned.  But what I had planned can perhaps be improved.

Based upon the workflow group meeting from about two weeks ago,
my plan was to have files that had headers with : 1)  the shebang, followed by  2) The new licencing header text , followed by 3) ,the old copyright header if one was already present. In the interest of uniformity, this would be the only "format", and files like this one that did not have the word "Copyright" was to be made conformant by removing and replacing the old licencing header. 

But if it should retain the FMS licensing information, I personally prefer to retain it or its equivalent, though the complete file headers wont be as uniform as planned. But thinking outside the box and trying to achieve all goals, perhaps option D) below is worth considering ?

Possibilities are :
A) Keep the file  as I have it,
B) Remove the new licencing header.
C) Remove the old licencing header.
D)  Have a second new licencing header for files like this , which is just like the other new header but says "This is part of the GFDL FMS and FRE-NCTools packages ..." . The old header would be removed in this case since FMS is mentioned.

Reply:
@bensonr , @underwoo @ceblanton 
Rusty, Seth and Chris. Do we have a consensus on this yet. I think options B) and D) might be the best.

Reply:
@ngs333 as it is a derivative work of FMS/mpp, I prefer to have it use our current license header from FMS.  I'm open to other ideas and if someone feels strongly that it should refer to only FRE-NCtools, then that's what we'll do.

Reply:
I have update this file and several others to use the license header from FMS.